UnitTesting-master/src/test/java/seleniumMail/Pages/InboxPage.java

```diff
@@ -2,35 +2,54 @@
 
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
+import seleniumMail.Helpers.WebDriverSingleton;
 
 import java.time.Duration;
 import java.util.NoSuchElementException;
 
 public class InboxPage {
 
-    private static By rightBoxList = By.cssSelector(".ns-view-right-box");
-    private static By username = By.cssSelector(".b-user");
+    @FindBy(css = ".ns-view-right-box")
+    WebElement rightBoxList;
+
+    @FindBy(css = ".b-user")
+    WebElement username;
+
+    @FindBy(css = ".user-account_left-name")
+    WebElement userIcon;
+
+    @FindBy(xpath = "//span[.='Log out']")
+    WebElement logoutBtn;
 
     private static WebDriver driver;
 
-    public InboxPage(WebDriver driver) {
-        this.driver = driver;
+    public InboxPage() {
+        this.driver = WebDriverSingleton.getInstance().getDriver();
```

Comment:
You missed one important line in your page objects' constructors. Your page factory will not work